src/applications/vaos/tests/mocks/setup.js

```diff
@@ -71,6 +72,34 @@ export function createTestStore(initialState) {
   );
 }
 
+/**
+ * Creates a Redux store with the VAOS reducers loaded and the thunk middleware applied
+ * and overriding with a custom user profile reducer.
```

Comment:
I don't think we need this custom Redux store

Reply:
@jenniemc  - same thing as previous comment. The store isn't created in it entirety to support the test.